bots/images/scripts/rhel.setup

```diff
@@ -97,7 +97,7 @@ EOF
 
     elif [ "${IMAGE#rhel-8*}" != "$IMAGE" ]; then
         case "$IMAGE" in
-            rhel-8-0) REPO="latest-RHEL-8.0" ;;
+            rhel-8-0) REPO="latest-RHEL-8.0.1" ;;
```

Comment:
My intent was that "latest-RHEL-8.0" would always point to the nightlies for the upcoming 8.0.z, and that we *don't* want to hardcode the .1. If that is not what it is, and there is no such link, then fine.